scripts/install/install-dependencies.sh

```diff
@@ -574,7 +574,10 @@ function maybe_install_conda() {
 function get_installed_astrometry_version() {
   local -r solve_field="${ASTROMETRY_DIR}/bin/solve-field"
   if [[ -x "${solve_field}" ]] ; then
-    "${solve_field}" --help|(grep -E '^Revision [0-9.]+,' || /bin/true)|cut -c10-|cut -d, -f1
+    ("${solve_field}" --help \
```

Comment:
I use `$SOLVE_FIELD` elsewhere to point to the `solve-field` executable. Here it is just checking a revision number, correct? (Also, I keep meaning to submit a PR to astrometry.net to add a simple `--version` flag. Very annoying it doesn't have one.)

Reply:
Yup, just checking version number.